packages/hub/README.md

```diff
@@ -52,23 +64,7 @@ await deleteRepo({ repo, credentials });
 
 When uploading large files, you may want to run the `commit` calls inside a worker, to offload the sha256 computations.
 
-Also, use `Blob` to avoid loading the whole files in RAM. In `Node`, you can use a local `URL` for loading large files: 
-
-```ts
-import { pathToFileURL } from "node:url";
-
-await commit({
-  repo,
-  credentials,
-  operations: [
-    {
-      operation: "addOrUpdate",
-      path: 'large-file.bin',
-      content: pathToFileURL("./large-file.bin"),
-    },
-  ],
-});
-```
+Web resources and local files should be passed as `URL` whenever it's possible so they will be lazy loaded in chunks to reduce RAM usage.
```

Comment:
```suggestion
Remote resources and local files should be passed as `URL` whenever it's possible so they can be lazy loaded in chunks to reduce RAM usage. Passing a `File` inside the browser's context is fine, because it natively behaves as a `Blob`.
```